### ISSUE TITLE
Forcing in `Isotropic.py`

### ISSUE THREAD

**Issue by msmerlak** [spectralDNS/spectralDNS#28]
I understand you're following Lamorgese et al 2005, with a forcing term proportional to `U_hat` times a mask that cuts off high-k modes. However, I'm having a hard time understanding how you've implemented that in your integration routine. Line 239 (`c.U_hat *= (alpha*c.mask + (1-c.mask))`) is that forcing term, but where is it added to the RNS of the Navier-Stokes equation? 

As a test that forcing is consistent, I changed the dissipation scale `kd` and checked whether `eps` converges to 1, as I think it should when stationarity is reached. But it doesn't. What am I missing? 

Thanks for your help.

**Comment by mikaem** [comment]
The forcing is not added implicitly to the NS. The energy is corrected at the end of a time step, like with an operator splitting routine, using [line 210](https://github.com/spectralDNS/spectralDNS/blob/72f46bf6177d809cf0594ee515f3923c39df76ff/demo/Isotropic.py#L210) in master. Here we modify the `Kf2` lowest wavenumbers, that are now set to the 3 lowest, to obtain the correct energy. The correct energy is determined and computed on initialization (the `target` line 76). Note that the operator splitting is perfectly ok since the forcing term is linear. So I'm not forcing any `eps` to unity, I'm simply forcing a constant (and correct) energy. If everything is scaled as it should, then I think `eps` should go to unity, but it has been a while since I looked at this, so it may something I'm forgetting.

**Comment by msmerlak** [comment]
I see, you're fixing the turbulent energy kinetic energy `k` via the Taylor Reynolds number `Re_lam`, whereas Lamorgese et al. fix the energy injection rate `P=1` and measure `Re_lam` from the flow. Would be nice if `eps` converged to unity in both cases, though, for consistency.

**Comment by mikaem** [comment]
Yes, it certainly would. I think my approach is easier with the currently implemented solvers, so I went for it. Shouldn't be too hard to fix the injection rate instead, though. Would be grateful for any PR in that direction :-) I have been meaning to add better integrators, with implicit linear terms, and this forcing term would then fall into place easily.
